SourceX/DiabloUI/diabloui.cpp

```diff
@@ -303,7 +303,7 @@ void UiFocusNavigationEsc()
 		UiTextInputLen = 0;
 	}
 	if (gfnListEsc)
-		gfnListEsc(SelectedItem);
+		gfnListEsc();

```

Comment:
Please also correct https://github.com/diasurgical/devilutionX/blob/22a3657ee47c5f757a028b4bcd82cd056866c05c/SourceX/DiabloUI/diabloui.cpp#L18 :)

Reply:
The function signature is updated in the same commit or do you mean something different here?

Reply:
Arh missed that